src/commands/plugins/inspect.ts

```diff
@@ -0,0 +1,165 @@
+import * as path from 'path'
+import {Command, flags} from '@oclif/command'
+import {Plugin} from '@oclif/config'
+import * as chalk from 'chalk'
+import {exec} from 'child_process'
+import * as fs from 'fs-extra'
+
+import Plugins from '../../plugins'
+import {sortBy} from '../../util'
+
+const TAB = '  '
+
+type Dependencies = {
+  [key: string]: unknown;
+  from?: string;
+  version?: string;
+  name?: string;
+  dependencies: {
+    [key: string]: Dependencies;
+  };
+}
+
+export default class PluginsInspect extends Command {
+  static description = 'inspects an installed plugin';
```

Comment:
thinking of what a little more elaboration here might say: 'displays package properties of an install plugin'